src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -155,10 +155,334 @@ public SyntaxNode MethodDeclaration(IMethodSymbol method, IEnumerable<SyntaxNode
         }
 
         /// <summary>
-        /// Creates a method declaration.
+        /// Creates an implicit conversion declaration.
+        /// </summary>
+        public virtual SyntaxNode ImplicitConversionDeclaration(
+            IEnumerable<SyntaxNode> parameters = null,
+            SyntaxNode returnType = null,
+            Accessibility accessibility = Accessibility.NotApplicable,
+            DeclarationModifiers modifiers = default(DeclarationModifiers),
+            IEnumerable<SyntaxNode> statements = null)
+        {
+            throw new NotImplementedException();
```

Comment:
Hrmm.. I don't think so.  These would preferably just be abstract methods.  But we can't add abstract methods to our shipped types.  So these are virtual, and i think it's correct to say that if you hit this it was because they were not implemented yet.  As opposed to specific methods which a particular language might say were not supported to be implemented at all.